BurnerB/Questioner-ADC-
# Bg fix meetups endpoints 163143302

#### What does this PR do?
FIx meetups endpoint
#### Description of Task to be completed?
Fix meetups ,should respond with the specified data in challenge 2

#### How should this be manually tested?
clone this repo
CD into it
Set up a virtual environment by:
`sudo install virtualenv env`
then `source venv/bin/activate`

install all modules in requirements.txt
run flask by:
`FLASK_APP=run.py`
`FLASK_ENV =development`
`Flask run`
The app should now be running
copy the  routes :
`POST /meetups`
`GET /meetups/<int:meetup_id>`
`GET /meetups/upcoming`
`POST /meetups/<int:meetup_id>/rsvps`

Paste the route in postman and test 

#### What are the relevant pivotal tracker stories?
#163143302
#### Screenshots (if appropriate)
![rsvp](https://user-images.githubusercontent.com/45785786/51075554-07062d80-169e-11e9-9a69-661ffc7621ba.png)
![get specific meetup](https://user-images.githubusercontent.com/45785786/51075555-09688780-169e-11e9-82ff-15d62e84155a.png)
![add meetup](https://user-images.githubusercontent.com/45785786/51075556-0a011e00-169e-11e9-89b9-783ed853c1ec.png)
![get upcoming](https://user-images.githubusercontent.com/45785786/51075557-0bcae180-169e-11e9-9834-946cbee42409.png)
